ui/helpers/constants/routes.ts

```diff
@@ -160,6 +158,9 @@ const PATH_NAME_MAP = {
     'Decrypt Message Request Page',
   [`${CONFIRM_TRANSACTION_ROUTE}/:id${ENCRYPTION_PUBLIC_KEY_REQUEST_PATH}`]:
     'Encryption Public Key Request Page',
+  ///: BEGIN:ONLY_INCLUDE_IN(mmi)
+  [INSTITUTIONAL_FEATURES_DONE_ROUTE]: 'Institutional features done',
```

Comment:
Looks good to me! Just a question: Shouldn't this be like this to keep it in sync with the rest?
```suggestion
  [INSTITUTIONAL_FEATURES_DONE_ROUTE]: 'Institutional features done Page',
```